Dockerfile.cpu

```diff
@@ -38,7 +38,7 @@ ENV LANG=C.UTF-8
 
 COPY requirements.txt .
 COPY requirements_test.txt .
-COPY scripts/ scripts/
+COPY scripts/install_requirements.sh scripts/install_requirements.sh
```

Comment:
Can we keep copying this whole directory - using the pylint and mypy scripts in here make our dev environment, including getting CI to pass, exactly reproducible.